server/src/concepts/user/repositories/commands.js

```diff
@@ -0,0 +1,63 @@
+import User from "../model/User";
+
+const createUser = (data) => {
+  const newUser = new User({
+    ...data,
+  });
+
+  try {
+    newUser.save();
+    return newUser;
+  } catch (err) {
+    console.log(err);
+    return "error";
```

Comment:
We have to provide some specific message, we can do this here or in some higher layer, have this in mind please